web/src/pages/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import CodeBlock from '@theme/CodeBlock';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import styles from './index.module.css';
+import HomepageFeatures from '../components/HomepageFeatures';
+
+function HomepageHeader() {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <header className={clsx('hero hero--primary', styles.heroBanner)}>
+      <div className="container">
+        <h1 className="hero__title">{siteConfig.title} 👧</h1>
+        <p className="hero__subtitle">{siteConfig.tagline}</p>
+        <div className={styles.buttons}>
+          <Link
+            className="button button--secondary button--lg"
+            to="/docs/home">
```

Comment:
I don't think that is a good idea that the only way to open a documentation is to open it from `quick start` instead of `home` but the button says `quick start` so it should be `quick start`. We can also reconsider adding two buttons on the start page: home and quick-start but I do not insist on that.

```suggestion
            to="/docs/quick-start">
```